### PR TITLE
Show the extra declaration data in the task review screen

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: pages/task/read/views/project.jsx [hunk 1]
```diff
@@ -75,14 +76,42 @@ const Project = ({ task, project, establishment, children, schema, formFields })
...
+                    declarationConfirmed(declarations.awerb) &&
+                      <Fragment>
+                        <p>
+                          <Snippet>declarations.awerb.review-date</Snippet> {declarations['awerb-review-date']}
+                        </p>
+                      </Fragment>
+                  }
+                  {
+                    // we don't collect a reason for 'Not yet'
+                    declarations.awerb.toLowerCase() === 'no' &&
```
Comment (lennym) [inline pages/task/read/views/project.jsx]:
```suggestion
                    !declarationConfirmed(declarations.awerb) &&
```

Comment (lennym) [reply pages/task/read/views/project.jsx]: Oh wait, there's _three_ options now?

Comment (wheelsandcogs) [reply pages/task/read/views/project.jsx]: Yeah, that's why it's like that and not using the function

File: pages/task/read/content/project.js [hunk 2]
```diff
@@ -31,8 +31,18 @@ module.exports = {
...
+      name: 'PEL Holder:',
+      'endorsement-date': 'Endorsement date: '
+    },
+    'awerb': {
+      question: 'Has the application been reviewed by the AWERB of each relevant establishment?',
+      'review-date': 'AWERB review date:',
+      'no-review-reason': 'Reason'
+    },
+    'ready-for-inspector': {
+      question: 'Has the applicant deemed the application ready for Inspector approval?'
```
Comment (lennym) [inline pages/task/read/content/project.js]: The question text here seems to be different to the question text at https://github.com/UKHomeOffice/asl/pull/220/files#diff-3de8b1506fb6b5578f936c7788434911R25

Comment (wheelsandcogs) [reply pages/task/read/content/project.js]: Good catch, forgot to update it here.

File: pages/task/read/content/project.js [hunk 3]
```diff
@@ -31,8 +31,18 @@ module.exports = {
...
+      name: 'PEL Holder:',
+      'endorsement-date': 'Endorsement date: '
+    },
+    'awerb': {
+      question: 'Has the application been reviewed by the AWERB of each relevant establishment?',
+      'review-date': 'AWERB review date:',
+      'no-review-reason': 'Reason'
+    },
+    'ready-for-inspector': {
+      question: 'Has an inspector advised you that this version of your draft application is ready for assessment?'
```
Comment (lennym) [inline pages/task/read/content/project.js]: I'd be tempted to change this to remove the "you" and the "your" on account of the varying audiences for the page.

```suggestion
      question: 'Has an inspector advised that this version of the draft application is ready for assessment?'
```

**Pull request merged**